doc/nrf/releases/release-notes-2.0.1.rst

```diff
@@ -0,0 +1,104 @@
+.. _ncs_release_notes_201:
+
+|NCS| v2.0.1 Release Notes
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products in the nRF52, nRF53, and nRF91 Series.
+The SDK includes open source projects (TF-M, MCUboot, OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and re-distributed with the SDK.
+
+Release notes might refer to "experimental" support for features, which indicates that the feature is incomplete in functionality or verification, and can be expected to change in future releases.
+The feature is made available in its current state though the design and interfaces can change between release tags.
+The feature will also be labelled with "EXPERIMENTAL" in Kconfig files to indicate this status.
+Build warnings will be generated to indicate when features labelled EXPERIMENTAL are included in builds unless the Kconfig option :kconfig:option:`CONFIG_WARN_EXPERIMENTAL` is disabled.
+
+Highlights
+**********
+
+* Updated the :ref:`nrfxlib:ot_libs` in nrfxlib and made minor changes related to Thread.
```

Comment:
Might be worth mentioning that the changes affect/fix Thread 1.2 Certification?

Reply:
Yes, let's wait for Krzysztof's highlights first.

Reply:
Done.